.github/COMMIT_METADATA.md

```diff
@@ -21,6 +21,7 @@
 - cf63211ac313545dcd153a74e4d19d0be593db41 CLDR-16712 v44 CLDRModify
 - cbdd59767072c7ca154faae2d3b59ad6ad67dd1d CLDR-16650 v44 cherry-pick maint-43 to main
 - f534e312f9c5b6984e5a941746adff08eff66c50 CLDR-16577 v44 merge maint-43 to main
+- 09c8e10 CLDR-17277 v45 accidental merge commit
```

Comment:
This is different than the commit id I see in the commit checker run. Am I missing something?

Also why do we sometimes use the long vs the short numbers? It would be great if we could be consistent.

Reply:
wrong paste.  
Also i will switch it to the long hash.